api/src/main/resources/liquibase.xml

```diff
@@ -548,5 +548,17 @@
             <column name="initial_appointment_status" type="varchar(45)"/>
         </addColumn>
     </changeSet>
-
+    <changeSet id="202001021618" author="Sukreet">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM privilege WHERE privilege = 'Appointments: Add Providers';
```

Comment:
Will 'Appointments: Invite Providers' make more sense?

Reply:
@bsneha90  Sure, will update this PR

Reply:
Done